media-gfx/yacreader/yacreader-9.0.0.recipe

```diff
@@ -29,13 +32,14 @@ REQUIRES="
 	lib:libQt5Network$secondaryArchSuffix
 	lib:libQt5OpenGL$secondaryArchSuffix
 	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libunarr$secondaryArchSuffix
 	"
 
 BUILD_PREREQUIRES="
```

Comment:
`BUILD_PREREQUIRES` should come after `BUILD_REQUIRES`, see the [Guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)